tools/postgres/dbtool.js

```diff
@@ -0,0 +1,447 @@
+    console.log("Database already exists");
+  } else {
+    console.log(
+      safePsqlSpawn([
+        PG_CONFIG.urlWithoutDatabase,
+        "-tAc",
+        `CREATE DATABASE ${PG_CONFIG.database};`,
+      ]),
+    );
+  }
+
+  const doesSchemaExist = callPsql(
+    "SELECT schema_name FROM information_schema.schemata WHERE schema_name = 'webknossos';",
+  ).trim();
+  if (doesSchemaExist === "webknossos") {
+    console.log("Schema already exists");
+  } else {
+    refreshSchema();
+  }
+}
+
+function prepareTestDb() {
+  ensureDb();
+  refreshSchema();
+
+  const csvFolder = path.join(__dirname, "..", "..", "test", "db");
+  for (const filename of fs.readdirSync(csvFolder)) {
+    if (filename.endsWith(".csv")) {
+      console.log(`IMPORT ${filename}`);
+      safePsqlSpawn(
+        [
+          PG_CONFIG.url,
+          "-c",
+          `SET session_replication_role = replica; COPY webknossos.${filename.slice(
+            0,
+            -4,
+          )} FROM STDOUT WITH CSV HEADER QUOTE ''''`,
+        ],
+        {
+          input: fs.readFileSync(path.join(csvFolder, filename)),
+        },
+      );
+    }
+  }
+  console.log("✨✨ Done preparing test database");
+}
+
+function refreshSchema() {
+  console.log(safePsqlSpawn([PG_CONFIG.url, "-v", "ON_ERROR_STOP=ON", "-f", schemaPath]));
+}
+
+function dumpSchema(databaseUrl, schemaDir, silent = false) {
+  if (!fs.existsSync(schemaDir) || !fs.statSync(schemaDir).isDirectory) {
+    console.error("Schema directory $schemadir does not exist, aborting!");
+    process.exit(1);
+  }
+
+  for (const fn of fs.readdirSync(schemaDir)) {
+    fs.rmSync(path.join(schemaDir, fn), { recursive: true, force: true });
+  }
+
+  if (!silent) console.log(`Dumping database to ${schemaDir}.`);
+
+  const items = safePsqlSpawn([databaseUrl, "-c", "\\d+ webknossos.*"]).trimEnd();
+  for (const block of items.split("\n\n")) {
+    const [type, identifier] = block
+      .split("\n")[0]
+      .trim()
+      .split('"')
+      .map((s) => s.trim().replace(/"/g, "").replace(/ /g, ""));
+    if (!silent) console.log(type, identifier);
+    fs.writeFileSync(path.join(schemaDir, `${type}__${identifier}`), block + "\n");
+  }
+
+  const functions = safePsqlSpawn([databaseUrl, "-c", "\\df+ webknossos.*"]);
+  fs.writeFileSync(path.join(schemaDir, "Functions"), functions);
+
+  const schemaVersion = safePsqlSpawn([
+    databaseUrl,
+    "-tAc",
+    "SELECT schemaVersion FROM webknossos.releaseInformation;",
+  ]);
+  fs.writeFileSync(path.join(schemaDir, "schemaVersion"), schemaVersion);
+}
+
+function cleanSchemaDump(dumpDir) {
+  for (const filename of fs.readdirSync(dumpDir)) {
+    fs.writeFileSync(
+      path.join(dumpDir, filename),
+      fs
+        .readFileSync(path.join(dumpDir, filename), { encoding: "utf-8" })
+        .replace(/,$/gm, "")
+        .replace(/\\r/gm, "  ")
+        .split("\n")
+        .sort()
+        .join("\n"),
+    );
+  }
+}
+
+function dumpSchemaViaSqlFiles(filePaths) {
```

Comment:
`dumpExpectedSchema` would be clearer to me. The parameter could be named `sqlFilePaths` to convey the sql bit. Similarily, `dumpSchema` could be renamed to `dumpCurrentSchema`.